From 111d46fe7fcda9d3fd1985503ecd2f95611512f1 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Thu, 12 Oct 2017 14:02:53 +0800
Subject: [PATCH 2629/5242] MLK-13946-6: ARM64: dts: remove video-mode in hdmi
 audio node

commit  708c1fbef4175989ccd240781a939125675daa58 from
https://source.codeaurora.org/external/imx/linux-imx.git

according the new api, video-mode is not needed for hdmi audio, it is
replaced by protocol property.

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../arm64/boot/dts/freescale/fsl-imx8mq-evk-4k.dts |    6 ------
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts   |    3 +--
 2 files changed, 1 insertion(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-4k.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-4k.dts
index a040a65..37072ac 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-4k.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk-4k.dts
@@ -15,12 +15,6 @@
 
 #include "fsl-imx8mq-evk.dts"
 
-/ {
-	sound-hdmi {
-		video-mode = <97>;
-	};
-};
-
 &dcss {
 	disp-mode = <97>;	/* #97: 3840x2160p@60Hz 16:9 */
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index c89febc..b27bb5d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -73,8 +73,7 @@
 		compatible = "fsl,imx-audio-cdnhdmi";
 		model = "imx-audio-hdmi";
 		audio-cpu = <&sai4>;
-		video-mode = <16>;	/* #16: 1920x1080p@60HZ 16:9 */
-					/* #95: 3840x2160p@30Hz 16:9 */
+		protocol = <0>;
 	};
 
 	pwmleds {
-- 
1.7.9.5

